src/runtime-rs/Makefile

```diff
@@ -44,7 +44,14 @@ else
 endif
 
 ifeq ($(PREFIX),)
-    PREFIX := /usr
+PREFIX        := /usr
+EXEC_PREFIX   := $(PREFIX)/local
+##VAR BINDIR=<path> is a directory for installing executable programs
+BINDIR := $(EXEC_PREFIX)/bin
+else
+EXEC_PREFIX   := $(PREFIX)
+##VAR BINDIR=<path> is a directory for installing executable programs
+BINDIR := $(EXEC_PREFIX)/runtime-rs/bin/
```

Comment:
That's because when creating the kata-deploy image, the default installation path for go runtime is `$(EXEC_PREFIX)`/bin. That's why I put the rust runtime at this poistion to support multiple runtime.

Reply:
Suggest adding a comment.

Reply:
fixed